modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -839,10 +953,14 @@ void TFImporter::populateNet(Net dstNet)
                 layerParams.set("pad_w", paddings.at<float>(2));
 
                 CV_Assert(next_layers.size() == 1);
```

Comment:
Maybe we should replace all similar `CV_Assert` with `CV_CheckEQ`?

Reply:
In general - Yes.
Not sure that this should be a part of this PR (it is large enough and the main change here is refactored layers processing and errors handling).